src/SprykerEco/Yves/FactFinderWebComponents/Theme/default/views/pushed-products-block/pushed-products-block.twig

```diff
@@ -0,0 +1,24 @@
+{% extends template('widget') %}
+
+{% block body %}
+    <ff-campaign-pushed-products {{ _widget.config|raw }}>
+        <span id="productCampaignCaption">{{ 'factfinder.web-components.pushed.products.title' | trans }}</span>
+        <ff-record-list>
+            <!-- in this demo a click on the product sets the recommendation product-->
+            <ff-record>
+                {% verbatim %}
+                <img data-image="{{record.ImageName}}">
+                {% endverbatim %}
+                <!--<img data-image>-->
+                <div>
+                    {% verbatim %}
+                    <a href="https://www.bergfreunde.de/{{record.Deeplink}}" data-action="redirect">
```

Comment:
?

Reply:
That was done because FactFinder need to have that elements as is, but twig trying to process such code when see it, e.g. {{record.Deeplink}}. That's why it was wrapped with verbatim tag

Reply:
Sorry, I confusion was about to the URL, not anything else